openid-4-verifiable-presentations-1_0.md

```diff
@@ -3193,6 +3258,13 @@ in the IANA "Uniform Resource Identifier (URI) Schemes" registry [@IANA.URI.Sche
 * Change Controller: OpenID Foundation Digital Credentials Protocols Working Group - openid-specs-digital-credentials-protocols@lists.openid.net
 * Reference: (#openid4vp-scheme) of this specification
 
+## JSON Web Token Claims Registration
+
+* Claim Name: "ld"
```

Comment:
I look forward to recusing myself when this registration request makes it to IANA/DEs.